dolphinscheduler-api/src/test/java/org/apache/dolphinscheduler/api/controller/ResourcesControllerTest.java

```diff
@@ -171,7 +172,7 @@ public void testDownloadResource() throws Exception {
         MultiValueMap<String, String> paramsMap = new LinkedMultiValueMap<>();
         paramsMap.add("id", "5");
 
-        MvcResult mvcResult = mockMvc.perform(get("/resources/download")
+        MvcResult mvcResult = mockMvc.perform(get("/resources/5/download")
```

Comment:
please use args， like  get("/resources/{id}/download", 5)

Reply:
Update for this unit test.